views/stats.ejs

```diff
@@ -790,36 +832,35 @@ const metaDescription = getMetaDescription()
                     <%= parseFloat(weight.total.toFixed(2)).toLocaleString() %>
                   </span>
                 ">
-                  <span class="stat-name"><%= weight.name %>: </span>
-                  <span class="stat-value"><%= parseFloat(Math.round(weight.total)).toLocaleString() %></span
-                ></span>
-              </div>
-            <%
+            <span class="stat-name"><%= weight.name %>: </span>
+            <span class="stat-value"><%= parseFloat(Math.round(weight.total)).toLocaleString() %></span></span>
+        </div>
+        <%
           }
         %>
       </div>
 
       <div id="skill_levels_container">
         <% if ('levels' in calculated) { %>
```

Comment:
a lot of the indenting looks ok but if you could make statements like this have an indent that would be nice

Reply:
>statements like this

meaning things that would be indented in javascript like `if`, `for`, etc

Reply:
You mean `<% if (BLAH) { %>` or?